func/demo/ffmpeg_check.c

```diff
@@ -0,0 +1,29 @@
+#include <libavformat/avformat.h>
+
```

Comment:
Could this be C++ and not C? It's not a major issue, but in general we should write functions in C++ wherever possible. It makes it easier to use other shared libraries/ utilities, and makes the functions easier to write.